WordPress/WordPressTest/WKCookieJarTests.swift

```diff
@@ -3,7 +3,7 @@ import WebKit
 @testable import WordPress
 
 class WKCookieJarTests: XCTestCase {
-    var wkCookieStore = WKWebsiteDataStore.nonPersistent().httpCookieStore
+    var wkCookieStore: WKHTTPCookieStore!
```

Comment:
The value is being assigned in `setUp`. It's best to do it like that anyway, so we get a fresh one for every test.